mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2299,7 +2299,7 @@ where
         spent_by_me: big_decimal_from_sat_unsigned(spent_by_me, coin.as_ref().decimals),
         my_balance_change: big_decimal_from_sat(received_by_me as i64 - spent_by_me as i64, coin.as_ref().decimals),
         total_amount: big_decimal_from_sat_unsigned(input_amount, coin.as_ref().decimals),
-        tx_hash: tx.hash().reversed().to_vec().into(),
+        tx_hash: format!("{:02x}", BytesJson(tx.hash().reversed().to_vec())),
```

Comment:
What do you think about adding a trait like
```rust
pub trait ToTxHash {
  fn to_tx_hash(&self) -> String;
}
```
The trait can be implemented for `Vec<u8>` or even for `H256`

Reply:
> What do you think about adding a trait like
> 
> ```rust
> pub trait ToTxHash {
>   fn to_tx_hash(&self) -> String;
> }
> ```
> 
> The trait can be implemented for `Vec<u8>` or even for `H256`

I was thinking about this one too, will implement and replace every occurences!

Reply:
Great, thank you!

Reply:
Could you please check if you replaced `format!("{:02x}")` wherever possible? I found a few places :)